server/configmodule.py

```diff
@@ -82,6 +82,13 @@ class StanfordConfig(CustomConfig):
   BASE_HTML_PATH = 'custom_dc/stanford/base.html'
 
 
+class StanfordStagingConfig(CustomConfig):
+  NAME = "Google Stanford Data Commons (Staging)"
+  ENV_NAME = 'STANFORD-STAGING'
+  ENABLE_BLOCKLIST = True
+  BASE_HTML_PATH = 'custom_dc/stanford/base.html'
+
+
```

Comment:
Let's inherit from CustomConfig and StanfordConfig (similar to LocalStanfordConfig below). We shouldn't add a new ENV_NAME for this for simplicity (so all `stanford*` instances should behave the same way). The only override here should be for the mixer endpoint, API_PROJECT and SECRET_PROJECT

Reply:
Done